### ISSUE TITLE
500 error when I tried to sort by language

### ISSUE THREAD

**Issue by summasmiff** [chaserx/wheelhouse#17]
Wanted to see how I stacked up, so I clicked the 'Ruby' badge by my name to be taken to this url: 
`https://gitlingo.herokuapp.com/organizations/cirrusmio?sort=Ruby` and was presented with a 500 error

**Comment by chaserx** [comment]
oh! hmm.

**Comment by chaserx** [comment]
``` shell
PG::InvalidColumnReference: ERROR:  for SELECT DISTINCT, ORDER BY expressions must appear in select list
LINE 1: ...s_organizations"."organization_id" = $1  ORDER BY languages....
                                                             ^
: SELECT DISTINCT "members"."id" AS t0_r0, "members"."name" AS t0_r1, "members"."github_login" AS t0_r2, "members"."github_id" AS t0_r3, "members"."avatar_url" AS t0_r4, "members"."profile_url" AS t0_r5, "members"."last_fetched_at" AS t0_r6, "members"."created_at" AS t0_r7, "members"."updated_at" AS t0_r8, "languages"."id" AS t1_r0, "languages"."name" AS t1_r1, "languages"."bytes" AS t1_r2, "languages"."member_id" AS t1_r3, "languages"."created_at" AS t1_r4, "languages"."updated_at" AS t1_r5 FROM "members" LEFT OUTER JOIN "languages" ON "languages"."member_id" = "members"."id" INNER JOIN "members_organizations" ON "members"."id" = "members_organizations"."member_id" WHERE "members_organizations"."organization_id" = $1  ORDER BY languages.name = 'Ruby' DESC,
                                     languages.bytes DESC
```

**Comment by chaserx** [comment]
I think this has something to do with adding `uniq` to the HABTM association

**Comment by chaserx** [comment]
fixed by #18
